app/utils/user/index.ts

```diff
@@ -407,3 +407,19 @@ export const sortCustomProfileAttributes = (a: CustomAttribute, b: CustomAttribu
     const orderB = b.sort_order ?? Number.MAX_SAFE_INTEGER;
     return orderA === orderB ? a.name.localeCompare(b.name) : orderA - orderB;
 };
+
+/**
+ * Converts an array of custom profile attributes to a map of attributes by their id.
+ * @param convertedAttributes - The array of custom profile attributes to convert
```

Comment:
Nit: Then the variable should be called `attributesToConvert`. `convertedAttributes` would be the return value.